tests/manage/ocs_288.py

```diff
@@ -0,0 +1,292 @@
+        assert POOL.delete(resource_name=item)
+    return True
+
+
+def create_cephfilesystem():
+    """
+    Function for deploying CephFileSystem (MDS)
+    :param kwargs:
+    :return:
+    """
+    fs_data = defaults.CEPHFILESYSTEM_DICT.copy()
+    fs_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'cephfs'
+    )
+    fs_data['metadata']['namespace'] = ENV_DATA['cluster_namespace']
+    global CEPHFS_OBJ
+    CEPHFS_OBJ = OCS(**fs_data)
+    CEPHFS_OBJ.create()
+    assert POD.wait_for_resource(
+        condition='Running', selector='app=rook-ceph-mds'
+    )
+    assert validate_cephfilesystem(fs_name=fs_data['metadata']['name'])
+    return True
+
+
+def validate_cephfilesystem(fs_name):
+    """
+    Function for validating CephFileSystem Creation
+    :param fs_name:
+    :return:
+    """
+    tools_pod = pod.get_ceph_tools_pod()
+    ceph_validate = False
+    k8s_validate = False
+    cmd = "ceph fs ls"
+    log.info(fs_name)
+    out = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
+    if out:
+        out = out[0]['name']
+        log.info(out)
+        if out == fs_name:
+            log.info("FileSystem got created from Ceph Side")
+            ceph_validate = True
+        else:
+            log.error("FileSystem was not present at Ceph Side")
+            return False
+    result = CFS.get(resource_name=fs_name)
+    if result['metadata']['name']:
+        log.info(f"Filesystem got created from kubernetes Side")
+        k8s_validate = True
+    else:
+        log.error("Filesystem was not create at Kubernetes Side")
+        return False
+    return True if (ceph_validate and k8s_validate) else False
+
+
+def create_multiple_rbd_storageclasses(count=1, ):
+    """
+    Function for creating multiple rbd storageclass
+    :param count:
+    :return:
+    """
+
+    for sc_count in range(count):
+        log.info("Creating CephBlockPool")
+        pool_obj = helpers.create_ceph_block_pool()
+        helpers.create_storage_class(
+            constants.CEPHBLOCKPOOL,
+            interface_name=pool_obj.name,
+            secret_name=RBD_SECRET_OBJ.name
+        )
+
+    return True
+
+
+def get_cephblockpool():
+    sc_obj = POOL.get()
+    sample = sc_obj['items']
+    pool_list = [
+        item.get('metadata')['name'] for item in sample
+    ]
+    return pool_list
+
+
+def get_secret():
+    sc_obj = SECRET.get()
+    sample = sc_obj['items']
+    conside_secret = "csi-"
+    secret_list = [
+        item.get('metadata')['name'] for item in sample if (
+            conside_secret in item.metadata.name
+        )
+    ]
+    return secret_list
+
+
+def get_pvc():
+    sc_obj = PVC.get()
+    sample = sc_obj['items']
+
+    pvc_list = [
+        item.get('metadata')['name'] for item in sample
+    ]
+    return pvc_list
+
+
+def get_storageclass():
+    """
+    Function for getting all storageclass
+    :return:
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+
+    ignore_sc = 'gp2'
```

Comment:
create a const for it in ocs/constants.py

Reply:
done